neuralprophet/forecaster.py

```diff
@@ -1372,7 +1406,14 @@ def create_df_with_events(self, df, events_df):
         df = df_utils.return_df_in_original_format(df_created, received_ID_col, received_single_time_series)
         return df
 
-    def make_future_dataframe(self, df, events_df=None, regressors_df=None, periods=None, n_historic_predictions=False):
+    def make_future_dataframe(
+        self,
+        df: pd.DataFrame,
+        events_df: Optional[pd.DataFrame] = None,
+        regressors_df: Optional[pd.DataFrame] = None,
+        periods: Optional[int] = None,
+        n_historic_predictions: Union[bool, int] = False,
```

Comment:
We should rethink in the future if we allow int variables like `n_historic_predictions` to have bool default. Is there a best practise how to prevent that?

Reply:
We should make our default parameter selection more consistent. Currently we have `None`, `False`, `auto`, etc - but not related to this PR.